rpc/swagger/swagger.yaml

```diff
@@ -1615,112 +1616,121 @@ components:
                     example: "not enough gas"
                   info:
-                    type: "string"
+                    type: string
                     example: ""
                   gas_wanted:
-                    type: "string"
+                    type: string
                     example: "100"
                   gas_used:
-                    type: "string"
+                    type: string
                     example: "100"
                   events:
                     type: "array"
-                    x-nullable: true
+                    nullable: true
                     items:
                       type: "object"
                       properties:
                         type:
-                          type: "string"
+                          type: string
                           example: "app"
                         attributes:
                           type: "array"
-                          x-nullable: false
+                          nullable: false
                           items:
                             type: "object"
                             properties:
                               key:
-                                type: "string"
+                                type: string
                                 example: "Y3JlYXRvcg=="
                               value:
-                                type: "string"
+                                type: string
                                 example: "Q29zbW9zaGkgTmV0b3dva28="
+                              index:
+                                type: boolean
+                                example: true
                   codespace:
-                    type: "string"
+                    type: string
                     example: "ibc"
             begin_block_events:
               type: "array"
-              x-nullable: true
+              nullable: true
               items:
                 type: "object"
                 properties:
                   type:
-                    type: "string"
+                    type: string
                     example: "app"
                   attributes:
                     type: "array"
-                    x-nullable: false
+                    nullable: false
                     items:
                       type: "object"
                       properties:
                         key:
-                          type: "string"
+                          type: string
                           example: "Y3JlYXRvcg=="
                         value:
-                          type: "string"
+                          type: string
                           example: "Q29zbW9zaGkgTmV0b3dva28="
+                        index:
+                          type: boolean
+                          example: true
             end_block:
               type: "array"
-              x-nullable: true
+              nullable: true
               items:
                 type: "object"
```

Comment:
do we need to remove the quotation marks for these types?

Reply:
I was going to make it more consistent and refactor things in a different PR, but it works and renders correctly like this.